src/node_file.h

```diff
@@ -30,7 +30,7 @@ class FSContinuationData : public MemoryRetainer {
 
   uv_fs_t* req;
   int mode;
-  std::vector<std::string> paths;
+  std::vector<std::string> paths{};
```

Comment:
That seems like something we should address on the compiler side, rather than starting to fix up all of our code… it’s not like something bad is happening here

Reply:
The compiler can't tell if we want the default constructor or we forgot to initialize.
I'd rather be explicit, either here or in all the constructors.